### ISSUE TITLE
hmi-javascript async requests broke cross server getVar

### ISSUE THREAD

**Issue by LEvertz** [acplt/rte#21]
cross ov-server getVars result in an  infinite wait at the moment. 
If you use CSHMIModelHost in an fbVariableReference the getVar is send correctly and the answer comes back OK (tested with firebug) but it is never noticed. Instead the head row still shows it is loading (everything is happening in onLoad).
I went back through the old staticDisplayComponents back to revision 9695 of hmi-Javascript - yes, that's pre-github. Commit 9695 of hmi-JavaScript (and the follow up 9696 for the fbd in cshmi) are the first to show this error. Commit log of 9695 is:

9695 	405t 00h 	holger 	/develop/hmi/hmiJavaScript/trunk/JavaScript/ 	
async request gefixt und wieder aktiviert.
bugfix für timeevent cyctime=0

the only changed file seems to be
JavaScript/hmi-class-cshmi.js
but there could be earlier changes that were only activated with this commit.

Bottom line:
Since async-Requests are enabled referencing an fbVariable on a different ov-server on the same host does not work anymore.

**Comment by HolgerJeromin** [comment]
Probably this is a problem with the [detection code of the callback](https://github.com/acplt/rte/blob/master/addonlibs/hmi/hmiJavaScript/JavaScript/hmi-class-cshmi.js#L1058-L1070). You should add a breakpoint there for further debugging.

If i had to guess [this line](https://github.com/acplt/rte/blob/master/addonlibs/hmi/hmiJavaScript/JavaScript/hmi-class-cshmi.js#L6440) does not detect the correct ObjectPath in this case.

**Comment by hfaham** [comment]
Done!

**Comment by HolgerJeromin** [comment]
Good catch @hfaham !
